src/webgpu/api/validation/createBindGroupLayout.spec.ts

```diff
@@ -68,101 +70,83 @@ g.test('visibility_and_dynamic_offsets')
     }, !success);
   });
 
-g.test('min_buffer_binding_size')
+g.test('non_required_parameter_combined_with_type_separately')
```

Comment:
I refactored all non-required parameters but hasDynamicOffset (which is tested with visibility) and combined them with type separately. In addition, I added a separate test for multisample/viewDimension combination. And this patch https://github.com/gpuweb/cts/pull/242 works! Thank you.

Reply:
Looks great!